README.rst

```diff
@@ -65,25 +68,20 @@ by using the --sign parameter.
 Building tapinstall (optional)
 ------------------------------
 
-The build system supports building tapinstall.exe (a.k.a. devcon.exe). However
-the devcon source code in WinDDK does not build without modifications which
-cannot be made public due to licensing restrictions. For these reasons the
+The build system supports building tapinstall.exe (a.k.a. devcon.exe), but the
 default behavior is to reuse pre-built executables. To make sure the buildsystem
 finds the executables create the following directory structure under
 tap-windows6 directory:
 ::
   tapinstall
-  └── 7600
-      ├── objfre_wlh_amd64
-      │   └── amd64
-      │       └── tapinstall.exe
-      └── objfre_wlh_x86
-          └── i386
-              └── tapinstall.exe
-
-This structure is equal to what building tapinstall would create. Replace 7600
-with the major number of your WinDDK version. Finally call buildtap.py with
-"--ti=tapinstall".
+  ├── Release
+  │   └── devcon.exe
+  └── x64
+      └── Release
+          └── devcon.exe
+
+This structure is equal to what building tapinstall would create. Call
+buildtap.py with "--ti=tapinstall".
```

Comment:
Does the build_tapinstall method work with this PR applied?

Reply:
Yes it does. I'll have to double-check to make sure the ARM64 instruction change updates this section too.

Reply:
I tested this earlier, but I suspect I didn't correctly empty `dist`. I just pushed a fix.